src/js/containers/covid19/budgetCategories/BudgetCategoriesTableContainer.jsx

```diff
@@ -265,7 +367,6 @@ const BudgetCategoriesTableContainer = (props) => {
                     expandable
                     rows={results}
                     columns={renderColumns()}
-                    divider={props.subHeading}
```

Comment:
Did design decide to remove the sub-heading above the child rows?

Reply:
Nope, I'll add it back in!